build/imports-generator.js

```diff
@@ -0,0 +1,109 @@
+/* global Promise */
+
+const fs = require('fs');
+const path = require('path');
+const browserify = require('browserify');
+const mkdirp = require('mkdirp');
+const getDirName = require('path').dirname;
+const importsFile = path.join(
+	__dirname,
+	'..',
+	'lib',
+	'core',
+	'imports',
+	'index.js'
+);
+
+async function run() {
+	// get an object map of modules to be imported as key value pairs based on args
+	const moduleMap = parseArgsAsRequireMap(process.argv.slice(2));
+	if (!moduleMap) {
+		throw new Error('Ensure supplied arguments are correct.');
+	}
+
+	// generate `axe.imports` content
+	const fileContent = `
+	/**
+	* Namespace for imports which holds globals of external module.
+	* @namespace imports
+	* @memberof axe
+	*/
+	axe.imports = {
+		${Object.keys(moduleMap)
+			.reduce((out, key) => {
+				out.push(`'${key}': require('${moduleMap[key]}')`);
+				return out;
+			}, [])
+			.join(', ')}
+	};`;
+
+	// write file content to `lib/core/imports/index.js`
+	try {
+		await createFile(importsFile, fileContent);
+	} catch (error) {
+		throw new Error(
+			`Cannot create file axe.imports with require modules`,
+			error
+		);
+	}
+
+	// browserify `lib/core/imports/index.js`
+	browserify(importsFile).bundle(async (err, result) => {
```

Comment:
Are we sure this is going to work in all cases? What about node.js usage?

Reply:
Some of our tests ensure that, what gets injected into `axe.imports` is as expected. 

I have done some node testing locally. But we will ensure that it works before a release.

Reply:
How do you expect we'll "ensure that it works before a release"? Instead of waiting until release-time, can we perform those tests on your branch?

Reply:
Will discuss with @WilcoFiers.